KsApi/models/ExtendedProjectProperties.swift

```diff
@@ -64,6 +63,11 @@ public struct ProjectTabCategoryDescription {
   public var id: Int
 }
 
+public struct ProjectTabGenerationDisclosure: Equatable {
```

Comment:
I'm not sure I understand the question. Do you mean what we'd do if the backend wants to add another question and answer to this section? Because if so, I think we'd either just create another view like the two we have, or we could use enums with an associated string (enum value representing the question and associated value representing the user-provided string) to simplify the code a bit. It'd be similar to the way we currently handle the checkmark cell. Though hopefully by that point, this will be rewritten in swiftui!

Reply:
lol, sure. I was just thinking we'd just send an array of strings into this function and in the cell, do a `forEach` on each `String` to create the views.

Reply:
We'd still need a way to tell what question/title each string is associated with, so we can't just do an array. But let's shelve this for if it becomes relevant :)